tests/test_redirect.py

```diff
@@ -4,107 +4,77 @@
 from requests import request
 
 
-def test_redirect_status_code():
+def test_redirect_status_code(purge_all_links, create_shortcut_link):
     # precondition
-    request(
-        method='DELETE',
-        url='http://localhost:8888/admin/all_links',
-        data='{"Are you sure?":"Yes"}'
-    )
-    create = request(
-        method='POST',
-        url='http://localhost:8888/shortcut',
-        data='{ "link": "https://github.com/Yurasb/url_shortener_testing"}'
-    )
+    purge_all_links
+    create_shortcut_link
     # action
     response = request(
         method='GET',
-        url='http://localhost:8888/r/{}'.format(create.json()['id'])
+        url='http://localhost:8888/r/{}'.format(
+            create_shortcut_link.json()['id']
+        )
     )
     # validation
     assert response.status_code == 302
 
 
-def test_redirect_is_redirect():
+def test_redirect_is_redirect(purge_all_links, create_shortcut_link):
     # precondition
-    request(
-        method='DELETE',
-        url='http://localhost:8888/admin/all_links',
-        data='{"Are you sure?":"Yes"}'
-    )
-    create = request(
-        method='POST',
-        url='http://localhost:8888/shortcut',
-        data='{ "link": "https://github.com/Yurasb/url_shortener_testing"}'
-    )
+    purge_all_links
+    create_shortcut_link
     # action
     response = request(
         method='GET',
-        url='http://localhost:8888/r/{}'.format(create.json()['id'])
+        url='http://localhost:8888/r/{}'.format(
+            create_shortcut_link.json()['id']
+        )
     )
     # validation
     assert response.is_redirect
 
 
-def test_redirect_body():
+def test_redirect_body(purge_all_links, create_shortcut_link):
     # precondition
-    request(
-        method='DELETE',
-        url='http://localhost:8888/admin/all_links',
-        data='{"Are you sure?":"Yes"}'
-    )
-    create = request(
-        method='POST',
-        url='http://localhost:8888/shortcut',
-        data='{ "link": "https://github.com/Yurasb/url_shortener_testing"}'
-    )
+    purge_all_links
+    create_shortcut_link
     # action
     response = request(
         method='GET',
-        url='http://localhost:8888/r/{}'.format(create.json()['id'])
+        url='http://localhost:8888/r/{}'.format(
+            create_shortcut_link.json()['id']
+        )
     )
     # validation - to be completed using XML-schema
     assert response.content
 
 
-def test_redirect_wrong_method_status_code():
+def test_redirect_wrong_method_status_code(purge_all_links, create_shortcut_link):
     # precondition
-    request(
-        method='DELETE',
-        url='http://localhost:8888/admin/all_links',
-        data='{"Are you sure?":"Yes"}'
-    )
-    create = request(
-        method='POST',
-        url='http://localhost:8888/shortcut',
-        data='{ "link": "https://github.com/Yurasb/url_shortener_testing"}'
-    )
+    purge_all_links
+    create_shortcut_link
     # action
     response = request(
```

Comment:
You should import this things as:
`import requests`
or
`from requests import request`
and use the same in all code. It's called code style. =)